src/gui/widgets/rich_label.cpp

```diff
@@ -403,26 +375,23 @@ std::pair<config, point> rich_label::get_parsed_text(
 			// table layouting
 			row_y = prev_blk_height;
 			row_idx = 0;
-			for (config& row : tag.cfg.child_range("row")) {
+			for(const config& row : child.child_range("row")) {
 				col_x = 0;
 				col_idx = 0;
 				max_row_height = 0;
 
-				for (config& col : row.child_range("col")) {
-					config col_cfg;
-					config& col_txt_cfg = col_cfg.add_child("text");
-					col_txt_cfg.append_attributes(col);
-					col_cfg.append_children(col);
+				for(const config& col : row.child_range("col")) {
+					config col_cfg{"text", col};
```

Comment:
I just realized I misread the original code here. Only the *attributes* go in the `text` tag... Will fix.